packages/web-react/src/components/ModalComposed/README.md

```diff
@@ -0,0 +1,41 @@
+# Modal
+
+```jsx static
+import {
+  ModalComposed,
```

Comment:
This is ok for export. But you have badly named files and components within. File names, as they are know, are in conflict with simple Modal.